skops/io/tests/test_persist.py

```diff
@@ -804,3 +804,75 @@ def test_loads_from_str():
     msg = "Can't load skops format from string, pass bytes"
     with pytest.raises(TypeError, match=msg):
         loads("this is a string")
+
+
+class _BoundMethodHolder:
+    """Used to test the ability to serialize and deserialize bound methods"""
+
+    def __init__(self, object_state: str):
+        self.object_state = (
+            object_state  # Initialize with some state to make sure state is persisted
+        )
+        self.chosen_function = (
+            np.log  # bind some method to this object, could be any persistable function
+        )
```

Comment:
```suggestion
        # Initialize with some state to make sure state is persisted
        self.object_state = object_state
        # bind some method to this object, could be any persistable function
        self.chosen_function = np.log
```

May be more readable, WDYT?

Reply:
I 100% agree

I originally had these as just normal inline comments, but I think they got auto-linted by the pre-commit hook to these odd multiline bracket statements